src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -206,7 +207,7 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Update an assessment after a complaint was accepted.
+     * PUT text-submissions/:submissionId/assessment-after-complaint : Update an assessment after a complaint was accepted.
```

Comment:
the URL below can also be improved. If something is updated, then the id of the object to be updated should be given in the URL in my opinion. Also there is a "/" in the beginning.